python/paddle/incubate/nn/functional/__init__.py

```diff
@@ -45,4 +46,5 @@
     'variable_length_memory_efficient_attention',
     "fused_rms_norm",
     "fused_layer_norm",
+    "masked_multihead_attention",
```

Comment:
because of Integrating rotary_embedding, attention and etc, it should be named `fused_masked_multihead_attention` according to regulations